### PR TITLE
adds alternate forms flag to HCA landing page

### PR DESCRIPTION
**Pull request by crwallace**

### CODE REVIEW

File: _healthcare/apply.md [hunk 1]
```diff
@@ -53,7 +53,41 @@ If you served on active duty, and separated under conditions other than dishonor
...
-**Notes:**
+### Additional forms for caregivers and active duty military
+
+- [Update income from a previous application (10-10EZR)](/healthcare/forms/vha-10-10ezr-fill.pdf)
+- [Apply as active duty military (10-10AD)](https://www.1010ez.med.va.gov/sec/vha/1010ez/Form/1010ez.pdf)
+- [Apply as a caregiver (10-10CG)](/healthcare/forms/vha-10-10CG.pdf)
+- [Apply as National Guard and Reserve (10-10EZ-Demob)](/healthcare/forms/10-10EZ-DEMOB.pdf)
+- [Apply as a member of the Justice Outreach Program (10-10EZ-JO)](/healthcare/forms/10-10EZ-JO.pdf)
+
+<dl>
```
Comment (webinista) [inline _healthcare/apply.md]: Why didn't we use an ordered or unordered list here?

Comment (crwallace) [inline _healthcare/apply.md]: We got a quick styling benefit going with `dl`. No other reason.

**Comment by crwallace** [comment]
From slack:
```
tiffany brown [5:05 PM]  
added a comment. otherwise, it looks fine.
```

Merge rationale.

**Comment by webinista** [comment]
The `dl` thing bugs me, but that's minor and can be fixed later. Otherwise, LGTM.

**Pull request merged**